.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
is inteded to be empty?

Reply:
is added by husky, not sure about the meaning